test/test_lomap.py

```diff
@@ -209,138 +243,151 @@ def test_clip_on_3d(self):
     # Test disallowing turning a methyl group (or larger) into a ring atom
     def test_transmuting_methyl_into_ring_rule(self):
         logging.basicConfig(format='%(message)s', level=logging.CRITICAL)
-        testdata=[('phenyl.sdf','toluyl3.sdf',1),
-                 ('toluyl3.sdf','chlorotoluyl1.sdf',1),
-                 ('toluyl3.sdf','phenylfuran.sdf',math.exp(-0.1*4)),
-                 ('toluyl3.sdf','phenylpyridine1.sdf',math.exp(-0.1*4)),
-                 ('phenyl.sdf','phenylfuran.sdf',1),
-                 ('phenyl.sdf','phenylpyridine1.sdf',1),
-                 ('chlorophenol.sdf','phenylfuran.sdf',1)
+        testdata=[(_rf('transforms/phenyl.sdf'), _rf('transforms/toluyl3.sdf'), 1),
+                 (_rf('transforms/toluyl3.sdf'), _rf('transforms/chlorotoluyl1.sdf'), 1),
+                 (_rf('transforms/toluyl3.sdf'), _rf('transforms/phenylfuran.sdf'), math.exp(-0.1*4)),
+                 (_rf('transforms/toluyl3.sdf'), _rf('transforms/phenylpyridine1.sdf'), math.exp(-0.1*4)),
+                 (_rf('transforms/phenyl.sdf'), _rf('transforms/phenylfuran.sdf'), 1),
+                 (_rf('transforms/phenyl.sdf'), _rf('transforms/phenylpyridine1.sdf'), 1),
+                 (_rf('transforms/chlorophenol.sdf'), _rf('transforms/phenylfuran.sdf'), 1)
                  ]
         lg = RDLogger.logger()
         lg.setLevel(RDLogger.CRITICAL)
-        for d in testdata:
-            parent=Chem.MolFromMolFile('test/transforms/'+d[0],sanitize=False, removeHs=False)
-            comp=Chem.MolFromMolFile('test/transforms/'+d[1],sanitize=False, removeHs=False)
+        for fn1, fn2, expected in testdata:
+            parent=Chem.MolFromMolFile(fn1, sanitize=False, removeHs=False)
+            comp=Chem.MolFromMolFile(fn2, sanitize=False, removeHs=False)
```

Comment:
I may have missed the definitation of `fn`/`fn1/2`, but perhaps it would be better to replace with `f_name` or something similarly more verbose.